packages/google_sign_in/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -240,6 +240,7 @@ public void init(
                     "default_web_client_id", "string", registrar.context().getPackageName());
         if (clientIdIdentifier != 0) {
           optionsBuilder.requestIdToken(registrar.context().getString(clientIdIdentifier));
+          optionsBuilder.requestServerAuthCode(registrar.context().getString(clientIdIdentifier));
```

Comment:
I modified this to not request id token but only request server auth code, also reading clientId passed in dart constructor rather than from resources as these can be different

Reply:
It seems only gets context from registrar. With v2 embedding, we would need to get the context from the plugin binding.

Reply:
I know but is it better to use this PR to support v2 emmbed?
support v2 emmbed is not the same as @vishna opinion, is it?

Reply:
I know but is it better to use this PR to support v2 emmbed?
support v2 emmbed is not the same as @vishna opinion, is it?